test/unit/iam-token-manager.test.js

```diff
@@ -79,64 +77,48 @@ describe('iam_token_manager_v1', function() {
     done();
   });
 
-  it('should use a valid access token if one is stored', async done => {
+  it('should refresh an access token whose refreshTime has passed', async done => {
     const instance = new IamTokenManager({ apikey: 'abcd-1234' });
     const requestMock = jest.spyOn(instance, 'requestToken');
 
     const currentTokenInfo = {
-      access_token: ACCESS_TOKEN,
-      refresh_token: '5678',
-      token_type: 'Bearer',
-      expires_in: 3600,
-      expiration: Math.floor(Date.now() / 1000) + 3000,
+      access_token: CURRENT_ACCESS_TOKEN,
     };
 
     instance.tokenInfo = currentTokenInfo;
-    instance.timeToLive = currentTokenInfo.expires_in;
-    instance.expireTime = currentTokenInfo.expiration;
-
-    const token = await instance.getToken();
-    expect(token).toBe(ACCESS_TOKEN);
-    expect(requestMock).not.toHaveBeenCalled();
-    done();
-  });
+    instance.expireTime = Math.floor(Date.now() / 1000) + 60;
+    instance.refreshTime = Math.floor(Date.now() / 1000) - 1;
 
-  it('should refresh an access token without expires_in field', async done => {
```

Comment:
This test was actually no longer valid, since the `expires_in` field is not used. It only happened to pass because the test did not set the `expireTime` in the instance.